### ISSUE TITLE
Stuck at stage: configure_vm

### ISSUE THREAD

**Issue by ramondsq** [myspaghetti/macos-virtualbox#640]
OS: Windows 11 Pro 22H2 22621.1105
VirtualBox version: 7.0.6 r155176
<img width="583" alt="image" src="https://user-images.githubusercontent.com/57244337/213860518-341083b9-0f88-4b05-823e-63dad9fd5610.png">
VirtualBox logs: 
[VBoxHardening.log](https://github.com/myspaghetti/macos-virtualbox/files/10471853/VBoxHardening.log)
[VBox.log](https://github.com/myspaghetti/macos-virtualbox/files/10471854/VBox.log)

**Comment by myspaghetti** [comment]
I haven't tested the script on VirtualBox 7. It's possible `VBoxManage` outputs something to STDERR. Could you paste the output of the following commant?

```
VBoxManage modifyvm macOS --cpus 2 \
            --memory 4096 --vram 128 --pae on \
            --boot1 none --boot2 none --boot3 none --boot4 none \
            --firmware efi --rtcuseutc on --chipset ich9 \
            --mouse usbtablet --keyboard usb --audiocontroller hda \
            --audiocodec stac9221 --audio=none
```

Attaching the output file of `./macos-guest-virtualbox.sh troubleshoot` would be helpful too, as the error could be caused by type errors, for example if `cpu_count`, `memory_size`, or `gpu_vram` are strings instead of integers.

**Comment by jshin313** [comment]
@myspaghetti

Running the VBoxManage command gives: 

`Warning: --audio is deprecated and will be removed soon. Use --audio-driver instead!
`

Virtualbox 7.0.6 was released quite recently on  January 17 2023 and one of the changes is this deprecation message.

**Comment by Strato30** [comment]
I have this issue too.

**Comment by myspaghetti** [comment]
Seems to work [when removing the `--audio` flags](https://github.com/myspaghetti/macos-virtualbox/commit/f84343fc6bf15c59f1d3b8089ca22bee8b6859bb) on the one machine I tested it... So not very extensively tested but I'm going to say [fixed in version 0.99.2.2](https://github.com/myspaghetti/macos-virtualbox/blob/fec574030cd1db83578a0a8a554f8155dd25c394/macos-guest-virtualbox.sh)
